src/components/Roadmap.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import data from 'data.json';
+
+//styles
+import '../index.scss';
+
+//types
+import { Status, STATUSES } from 'types';
+
+export const Roadmap = () => {
+  let plannedCount = data.productRequests.filter((item) => {
+    return item.status === 'planned';
+  }).length;
+
+  let inProgressCount = data.productRequests.filter((item) => {
+    return item.status === 'in-progress';
+  }).length;
+
+  let liveCount = data.productRequests.filter((item) => {
+    return item.status === 'in-progress';
```

Comment:
Should this be `live` or something similar?

Reply:
Woops, yes it should be live. Changed